srcpkgs/ranger/template

```diff
@@ -3,21 +3,25 @@ pkgname=ranger
 version=1.9.3
 revision=4
 build_style=python3-module
-pycompile_module="ranger"
 hostmakedepends="python3"
 makedepends="python3-devel"
 depends="python3"
 checkdepends="python3-pytest flake8 ncurses-term"
 short_desc="File manager with an ncurses frontend written in Python"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="André L. C. Moreira <andrelcmoreira@disroot.org>"
 license="GPL-3.0-only"
 homepage="https://ranger.github.io"
+changelog="https://raw.githubusercontent.com/ranger/ranger/master/CHANGELOG.md"
 distfiles="${homepage}/${pkgname}-${version}.tar.gz"
 checksum=ce088a04c91c25263a9675dc5c43514b7ec1b38c8ea43d9a9d00923ff6cdd251
 
 do_check() {
 	# test_pylint is failing,
 	# As a distro, we don't want to check their code style
-	TERM=linux TERMINFO=/usr/share/terminfo \
+
+	# as workaround to fix this step, we set the 'PYTHON' variable due the lack
+	# of 'which' command (whose is used in the ranger's Makefile to find the
+	# suitable python version) in base-chroot.
+	PYTHON=python TERM=linux TERMINFO=/usr/share/terminfo \
```

Comment:
Does it work with `TERM=ansi`? or `TERM=xterm`?

Reply:
Also without `TERMINFO`?

Reply:
> Does it work with TERM=ansi? or TERM=xterm?

Yes.

> Also without TERMINFO?

Yes. Do you think we can remove it?I'm not sure about its relevance here. I took a quick look at the project's source code and I figured out ranger doesn't rely on this variable directly.